libs9s/s9soptions.cpp

```diff
@@ -7198,6 +7222,7 @@ S9sOptions::printHelpBackup()
 "  --delete-snapshot-repository   Deletes a snapshot repository on elastisearch cluster.\n"
 "\n"
 "  --backup-id=ID             The ID of the backup.\n"
+"  --backup-list=\"ID1, ID2\" The list of IDs of the backups.\n"
```

Comment:
Please fix indentation, this line looks odd when compared to others in cmdline (at least mine)
`
--backup-id=ID             The ID of the backup.
--backup-list="ID1, ID2" The list of IDs of the backups.
--cluster-id=ID            The ID of the cluster.

`